src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEPropertySymbol.cs

```diff
@@ -532,7 +532,19 @@ public override ImmutableArray<CSharpAttributeData> GetAttributes()
             if (_lazyCustomAttributes.IsDefault)
             {
                 var containingPEModuleSymbol = (PEModuleSymbol)this.ContainingModule;
-                containingPEModuleSymbol.LoadCustomAttributes(_handle, ref _lazyCustomAttributes);
+
+                ImmutableArray<CSharpAttributeData> attributes = containingPEModuleSymbol.GetCustomAttributesForToken(
+                      _handle,
+                      out _,
+                      this.RefKind == RefKind.RefReadOnly ? AttributeDescription.IsReadOnlyAttribute : default,
+                      out _,
+                      default,
+                      out _,
+                      default,
+                      out _,
+                      default);
```

Comment:
It looks like several uses of `GetCustomAttributesForToken()` take a single filter. Consider adding an overload for that case.